src/integrationTest/java/uk/gov/hmcts/reform/ccd/documentam/controller/CaseDocumentAmControllerIT.java

```diff
@@ -282,6 +282,25 @@ void shouldBeForbiddenWhenPatchingMetaDataOnDocumentWithIncorrectDocumentHashTok
                 body.getCaseId()));
     }
 
+    @Test
+    void shouldBeForbiddenWhenPatchingMetaDataOnDocumentWithIncorrectS2SService() throws Exception {
+
+        CaseDocumentsMetadata metadata = CaseDocumentsMetadata.builder()
+            .caseId(CASE_ID_VALUE)
+            .caseTypeId(CASE_TYPE_ID_VALUE)
+            .jurisdictionId(JURISDICTION_ID_VALUE)
+            .documentHashTokens(List.of(new DocumentHashToken(DOCUMENT_ID.toString(), "567890976546789")))
+            .build();
+
+        mockMvc.perform(patch(MAIN_URL + ATTACH_TO_CASE_URL)
+                            .headers(createHttpHeaders("xui_webapp"))
```

Comment:
Use existing constant `SERVICE_NAME_XUI_WEBAPP`

Reply:
Even better get rid of `SERVICE_NAME_XUI_APP` altogether from this class, and replace with references to static import of

`uk.gov.hmcts.reform.ccd.documentam.apihelper.Constants.XUI_WEBAPP`

Reply:
Constants is a main java class. So, removed from there and added to required tests. 
cleaned up here 522b82c73c043bdbb8dedaa797594f26c15599e8